src/control/cmd/dmg/pool.go

```diff
@@ -238,3 +252,26 @@ func poolDestroy(log logging.Logger, conns client.Connect, poolUUID string, forc
 
 	return err
 }
+
+func poolGetACL(log logging.Logger, conns client.Connect, poolUUID string) error {
+	req := &client.PoolGetACLReq{UUID: poolUUID}
```

Comment:
Looking at the drpc handling I didn't see any test for an invalid uuid format being passed in. Do we want to check for the uuid format here and reject it early if its not properly formed?

Reply:
Check out srv_drpc_tests.c:117. It's being checked in the handler on the C side. We could also check it earlier if you prefer, but it seems like it needs to be checked on the C side regardless.